.github/workflows/main.yml

```diff
@@ -37,6 +37,9 @@ jobs:
       - name: Install dependencies
         run: yarn install --immutable
 
+      - name: Check for unmet constraints (fix w/ "yarn constraints --fix")
+        run: yarn constraints
+
```

Comment:
Is it not necessary to install the _constraints_ plugin in advance?
Does the container already have it installed?

Reply:
Everything was already set up, we just forgot to run the check on CI.

https://github.com/commercetools/ui-kit/blob/16cf82b43db20da11be0612ea64c30ca4c23654a/.yarnrc.yml#L4-L5
https://github.com/commercetools/ui-kit/blob/main/constraints.pro